hs_core/templates/search/search.html

```diff
@@ -2,7 +2,8 @@
 {% load geoanalytics_tags pages_tags mezzanine_tags keyword_tags comment_tags hydroshare_tags %}
 {% block title %}Discovery{% endblock %}
 {% block main %}
-<div class="container" id="main">
+<div id="discover-load" style="display: none"></div>
```

Comment:
Also, graying out and disabling the entire screen as the request is performed hinders usability. I think the screen should be left enabled, but display the loading icon above the table. While the request is being processed the table should not change until it has been completed (more specifically, the items should not be removed). 

There is also an issue with how the map loads. The loading only starts when the tab is clicked. It takes quite some time to render and while this happens there should be a better layout (right now we have weirdly sized empty boxes) with message to the user indicating it is loading.